test/func_tests/gs_tests.cpp

```diff
@@ -710,6 +711,62 @@ int main(int argc, char *argv[])
 
     grpc_client.send_goalstate_stream(10, "32");
   }
+  return rc;
+}
+
+int main(int argc, char *argv[])
+{
```

Comment:
Checking that why we need a main function in a gtest file.

Reply:
This is because that gs_tests runs as a separate process, instead of being a part of the ACA.